src/estimation-kdl/CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ target_link_libraries(${libraryname} idyntree-core idyntree-kdl idyntree-sensors
 get_property(IDYNTREE_TREE_INCLUDE_DIRS GLOBAL PROPERTY IDYNTREE_TREE_INCLUDE_DIRS)
 target_include_directories(${libraryname} BEFORE PRIVATE ${IDYNTREE_TREE_INCLUDE_DIRS})
 
+if((${CMAKE_CXX_COMPILER_ID} MATCHES "GNU") OR (${CMAKE_CXX_COMPILER_ID} MATCHES "Clang"))
+    target_compile_options(${libraryname} PRIVATE "-Wno-deprecated")
```

Comment:
I have to check with @drdanz , but I would say: CMake 3.0 for the next release, then CMake 3.5 for the next one (i.e. the `devel` branch), because for Debian oldstable the support policy is: 
> Supported only for either 3 or 6 months after a stable release